src/test/shell/bazel/BUILD

```diff
@@ -202,6 +202,30 @@ sh_test(
     tags = ["local"],
 )
 
+sh_test(
+    name = "bazel_java15_test",
+    srcs = ["bazel_java15_test.sh"],
+    args = [
+        # --java_toolchain and --host_java_toolchain values
+        "@local_java_tools//:toolchain_jdk_15",
+        # java_tools zip to test
+        "src/java_tools_java15.zip",
+    ] + select({
+        # --javabase and --host_javabase values
+        "//src/conditions:darwin": ["@openjdk15_darwin_archive//:runtime"],
+        "//src/conditions:darwin_x86_64": ["@openjdk15_darwin_archive//:runtime"],
+        "//src/conditions:windows": ["@openjdk15_windows_archive//:runtime"],
+        "//src/conditions:linux_x86_64": ["@openjdk15_linux_archive//:runtime"],
+    }),
+    data = [
+        ":test-deps",
+        "//src:java_tools_java15_zip",
```

Comment:
and `"//src:java_tools_java11_zip"` here...

tools 11 will have java 14 and java 15

Reply:
Nice thank you, done.